packages/next-dashboard/src/components/Fader.js

```diff
@@ -17,8 +22,8 @@ function Fader<El: string = 'div'>({
   className,
   children,
   ...props
-}: Props<El>): Element<El> | null {
-  if (as == null) return null;
+}: Props<El>): Element<El> | Element<typeof Fragment> {
+  if (as == null) return <Fragment />;
```

Comment:
Can't we just specify `Node` as return type? I don't really see any point in being more specific than that (AFAICT it mostly makes it more annoying to refactor the component without really offering any benefits).

I assume there's a reason for this change, presumably something where this component is used treats `null` as an empty component and `<Fragment />` as no components at all? I'm not sure if this is the right way to fix that though, or if the call-site might be more appropriate, since that seems like an annoying assumption to make. How is it used?

Reply:
I'm still curious wrt `<Fragment />` :p  (but like, it doesn't matter much and can be merged anyway)

Reply:
Oh, it was a reminant from back when I tried a bunch of stuff, and trying things like ref forwarding for a feature, dropped that but forgot to revert back the fragment to null. Also, I marked this "resolved" because I changed it to be null/Node (hence the "outdated" tag)